module4/homework/appendNewRecipe/AppendNewRecipe.cpp

```diff
@@ -0,0 +1,63 @@
+#include "AppendNewRecipe.hpp"
+#include <algorithm>
+#include <fstream>
+#include <map>
+
+bool AppendNewRecipe(std::vector<std::string> steps,
+                     const std::list<std::string>& ingredients,
+                     const std::deque<std::pair<size_t, char>>& amount) {
+    std::fstream recipes("recipes.txt", recipes.out | recipes.app);
```

Comment:
I recommend making file path const variable instead of using magic "recipes.txt".

Reply:
*constexpr :)

Reply:
+1

Reply:
good tip :)